config/manager/.key-server.conf

```diff
@@ -1,3 +0,0 @@
-# NOTE: update the Key server address beofere
```

Comment:
beofere -> before

Reply:
The file is (re)moved, hence this comment does not apply :)

Reply:
Argh :-)